ksql-metastore/src/main/java/io/confluent/ksql/metastore/MetastoreUtil.java

```diff
@@ -137,13 +137,13 @@ private Schema getKsqlType(final String sqlType) {
   }
 
   private String getKsqlTypeInJson(final Schema schemaType) {
-    if (schemaType == Schema.INT64_SCHEMA) {
+    if (schemaType.type() == Schema.Type.INT64) {
```

Comment:
why not use switch statement here instead a bunch of `IFs`? To make a code style consistent with what has been done earlier in the PR.